core/sail/api/src/main/java/org/eclipse/rdf4j/sail/helpers/AbstractSailConnection.java

```diff
@@ -394,6 +395,29 @@ public final CloseableIteration<? extends Statement> getStatements(Resource subj
 		}
 	}
 
+	@Override
+	public final CloseableIteration<? extends Statement> getStatements(StatementOrder order, Resource subj, IRI pred,
+			Value obj, boolean includeInferred, Resource... contexts) throws SailException {
```

Comment:
This is at the SailConnection level. It might be better to have the `order` argument be after `includeInferred` and before `Resource... contexts`.